src/test/spec/faas.rs

```diff
@@ -39,7 +39,7 @@ async fn check_faas_handshake(vars: &[(&'static str, &str)]) -> Result<()> {
     let _tv = TempVars::set(vars);
 
     let client = Client::test_builder().build().await;
-    client.list_database_names(doc! {}, None).await?;
+    client.list_database_names().filter(doc! {}).await?;
```

Comment:
nit: can the filter here be omitted entirely? I'd suspect empty filter is equivalent to no filter set

Reply:
Oh, yup, good catch.  Rote translation...